script.js

```diff
@@ -1,76 +1,221 @@
     } else if (
-      (userChoice == "reverse scissors" && botChoice == "paper") ||
-      (userChoice == "reverse paper" && botChoice == "stone") ||
-      (userChoice == "reverse stone" && botChoice == "scissors")
+      userChoice == "reverse scissors" ||
+      userChoice == "reverse paper" ||
+      userChoice == "reverse stone"
     ) {
-      var result = "You lose.";
+      // Checks user choice against bot choice and declares a result
+      if (
+        (userChoice == "reverse scissors" && botChoice == "scissors") ||
+        (userChoice == "reverse paper" && botChoice == "paper") ||
+        (userChoice == "reverse stone" && botChoice == "stone")
+      ) {
+        var result = "It's a draw.";
+        drawRecord++;
+      } else if (
+        (userChoice == "reverse scissors" && botChoice == "paper") ||
+        (userChoice == "reverse paper" && botChoice == "stone") ||
+        (userChoice == "reverse stone" && botChoice == "scissors")
+      ) {
+        var result = "You lose.";
+        lossRecord++;
+      } else {
+        var result = "You win.";
+        winRecord++;
+      }
+    } // Tells user to enter correct inputs if user entry checks fail
+    else {
+      return "Please input 'scissors', 'paper', or 'stone' to play the game.";
+    }
+    gameRecord++;
+
+    //Performance check (nomral)
+    if (winRecord > lossRecord) {
+      performanceMesssage = "Good Job!";
+    } else if (lossRecord > winRecord) {
+      performanceMesssage = "Keep trying...";
     } else {
-      var result = "You win.";
+      performanceMesssage = "What a close game...";
     }
-  } // Tells user to enter correct inputs if user entry checks fail
-  else {
-    return "Please input 'scissors', 'paper', or 'stone' to play the game.";
-  }
 
-  //Returns result to the player
-  return (
-    "You chose " +
-    userChoice +
-    "." +
-    "<br><br>" +
-    "The bot chose " +
-    botChoice +
-    "." +
-    "<br><br>" +
-    result +
-    "<br><br>" +
-    "Now you can type 'scissors', 'paper', or 'stone' to play another round!" +
-    "<br><br>" +
-    "Or add 'reverse' before your input to try something different."
-  );
+    //Returns result to the player
+    if (gameType == "normal") {
+      return (
+        "You chose " +
+        userChoice +
+        "." +
+        "<br><br>" +
+        "The bot chose " +
+        botChoice +
+        "." +
+        "<br><br>" +
+        result +
+        "<br><br>" +
+        "Now you can type 'scissors', 'paper', or 'stone' to play another round!" +
+        "<br><br>" +
+        "Or add 'reverse' before your input to try something different." +
+        "<br><br>" +
+        "Hey, " +
+        userName +
+        " you have played " +
+        gameRecord +
+        " games so far. Your win:loss:draw figures is " +
+        winRecord +
+        ":" +
+        lossRecord +
+        ":" +
+        drawRecord +
+        "." +
+        performanceMesssage +
+        "."
+      );
```

Comment:
I would suggest using template literals  over quotations especially with as many variables as this. You can make it a lot shorter that way. for example: 

` "You chose " +
        userChoice +
        "." +
        "<br><br>" +
        "The bot chose " +
        botChoice +
        "."`
    
  Could instead be:
  `You chose ${userChoice}. \n\n The bot chose ${botChoice}.`
   (put ` at the front and back, unfortunately putting it here indents the code so you cant see it)